src/main/java/com/tabnine/lifecycle/BinaryNotificationsLifecycle.kt

```diff
@@ -44,7 +44,7 @@ class BinaryNotificationsLifecycle(
                                             o.actions
                                         )
                                     )
-                                    if (o.actions?.contains(OPEN_HUB_ACTION) ?: false) {
+                                    if (o.actions?.any { it is String && it == OPEN_HUB_ACTION } == true) {
```

Comment:
Why do we need to check the type?

Reply:
How come '==' is good enough here but in getTooltipText we need equals?

Reply:
it is kotlin, the `==` is value equality.
regarding the type check, you are right, ill remove it